backend/audit/file_downloads.py

```diff
@@ -0,0 +1,43 @@
+from django.conf import settings
+from django.http import Http404
+from django.shortcuts import get_object_or_404
+
+from boto3 import client as boto3_client
+from botocore.client import ClientError, Config
+
+from audit.models import ExcelFile, SingleAuditReportFile
+
+
+def get_filename(sac, file_type):
+    if file_type == "report":
+        file_obj = get_object_or_404(SingleAuditReportFile, sac=sac)
+        return f"singleauditreport/{file_obj.filename}"
+    else:
+        file_obj = get_object_or_404(ExcelFile, sac=sac, form_section=file_type)
```

Comment:
It is supposed to. For the first version, we're passing back the Excel docs so that auditors can review what they submitted. In the case of large audits, they often use tools to do this. So, we're passing the submitted objects back.

Reply:
The plumbing is in place for excel downloads, but it's effectively unreachable code - we currently only have [PdfDownloadView](https://github.com/GSA-TTS/FAC/blob/90115af14020c0e15fb06604b6590bb5b05f859f/backend/dissemination/views.py#L133-L145) which hardcodes "report" for `file_type`.

I missed the XLSX Workbook download link on the summary page - I see that in figma now. Should we update this PR to include XLSX downloads or push that to 1.5?

Reply:
Also, figma shows a single XLSX download link -- is that intended to return a zip of all workbooks?